server/auth/index.js

```diff
@@ -9,23 +9,19 @@ import { AuthenticationError } from "../errors";
 import auth from "../middlewares/authentication";
 import validation from "../middlewares/validation";
 import { Team } from "../models";
-import { requireDirectory } from "../utils/fs";
+import providers from "./providers";
 
 const log = debug("server");
 const app = new Koa();
 const router = new Router();
 
 router.use(passport.initialize());
 
-// dynamically load available authentication providers
-requireDirectory(__dirname).forEach(([{ default: provider, config }]) => {
```

Comment:
There were multiple places this require loop happened, now it happens once in `./providers` and is imported elsewhere